lms/djangoapps/course_structure_api/v0/serializers.py

```diff
@@ -10,6 +11,7 @@ class CourseSerializer(serializers.Serializer):
     """ Serializer for Courses """
     id = serializers.CharField()  # pylint: disable=invalid-name
     name = serializers.CharField(source='display_name')
+    subject = serializers.SerializerMethodField()
```

Comment:
It can't be replaced with ChoiceField quickly, because 'subject' is String field in MongoDB